src/popup/components/ui/Input.tsx

```diff
@@ -62,24 +65,20 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
     <div className="space-y-2">
       {label ? <Label htmlFor={id}>{label}</Label> : null}
       {description ? <p className="px-2 text-xs">{description}</p> : null}
-      <div className="relative">
-        {addOn ? (
-          <div
-            className={cn(
-              'pointer-events-none absolute inset-y-0 flex w-10 items-center justify-center text-sm font-medium',
-              addOnPosition === 'left' ? 'left-0' : 'right-0',
-            )}
-          >
-            {addOn}
+      <div className={cn('relative', wrapperClassName)}>
+        {leadingAddOn ? (
+          <div className="pointer-events-none absolute inset-y-0 left-0 flex w-10 items-center justify-center text-sm font-medium">
+            {leadingAddOn}
           </div>
         ) : null}
         <input
```

Comment:
`readOnly` is not passed as an attribute.

Reply:
Good catch!